Lib/test/support/__init__.py

```diff
@@ -1002,16 +1015,41 @@ def wrapper(self):
 #=======================================================================
 # unittest integration.
 
-def _id(obj):
-    return obj
+def mark(label, *, globals=None):
```

Comment:
I had a similar API in one of my OS libs, users didn't like it at all :(

I think it would be better to use `mark(..., *, module=False)`

This way we can:
1. Simplify the API for readers (even if our users are people working on CPython)
2. Use `frame` hack to get the needed `globals()`

Reply:
I do not like to depend on such hack it tests. If it does not work, many unrelated tests will not even be able to load.

Since it is an internal API used in limited number of places, simplicity for users is less important. `support.mark('gui', globals=globals())` is not much worse than `support.mark('gui', module=True)`